packages/dotcom-ui-client-embed/src/client/ClientEmbedData.ts

```diff
@@ -0,0 +1,15 @@
+export default class ClientEmbedData {
```

Comment:
Any way to add types here and then make them more specific when the class gets extended?